e2e/api_test.go

```diff
@@ -242,6 +244,92 @@ func TestE2E_StatusEndpoints(t *testing.T) {
 	delete()
 }
 
+func TestE2E_TaskEndpoints_UpdateEnableDisable(t *testing.T) {
```

Comment:
Just checking if I'm understanding the test correctly to make sure I'm reviewing with the right context.

`TestE2E_TaskEndpoints_UpdateEnableDisable` replaces the need for `TestE2EDisabledTask`.
1. The test starts off with a disabled task
2. API to enable it
3. API to disable it then trigger a task run by adding a new service instance to very no runs because it's disabled.

Reply:
Yes! Your understanding is correct. `TestE2EDisabledTask` does step 1. `TestE2E_TaskEndpoints_UpdateEnableDisable` needs to start off with step 1, so I combined the two tests. Let me add a comment at the top of the test with a summary similar to what you have here.

Reply:
The test summary comment is 💯 thank you!